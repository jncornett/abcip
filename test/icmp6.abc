d(stack="eth:ip6:icmp6:ip6:tcp")
a(icmp6:type=1, code=0)

