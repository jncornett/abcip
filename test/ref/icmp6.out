IP6 (flowlabel 0x00001, hlim 64, next-header ICMPv6 (58) payload length: 68) ::ffff:10.1.2.3 > ::ffff:10.9.8.7: [icmp6 sum ok] ICMP6, destination unreachable, length 68, unreachable route ::ffff:10.9.8.7
	0x0000:  0209 0807 0605 0201 0203 0405 86dd 6000  ..............`.
	0x0010:  0001 0044 3a40 0000 0000 0000 0000 0000  ...D:@..........
	0x0020:  ffff 0a01 0203 0000 0000 0000 0000 0000  ................
	0x0030:  ffff 0a09 0807 0100 7a30 0001 0001 6000  ........z0....`.
	0x0040:  0001 0014 0640 0000 0000 0000 0000 0000  .....@..........
	0x0050:  ffff 0a01 0205 0000 0000 0000 0000 0000  ................
	0x0060:  ffff 0a09 0807 bdec 0008 0000 0000 0000  ................
	0x0070:  0000 5000 2000 b3da 0000                 ..P.......
